README.md

```diff
@@ -412,6 +412,54 @@ Refer to our [profiling document](doc/profiling.md).
 
 ## Updating / Installing dependencies
 
+### Python
 To add a new dependency, use `pipenv install [package-name]`, which not only installs the package but Pipenv will also go to the trouble of updating the Pipfile as well.
 
 NB: both the Pipfile and Pipfile.lock files are required in source control to accurately pin dependencies.
+
+### JavaScript
+To add a new dependency, use `yarn add [package-name]` and just run `yarn` if you need to install all the packages needed for locally.
+
+
+## Testing design system changes (locally) without pushing to actual CDN
+
+### On DS Repo
+Checkout branch with new changes on
+
+Then in the console run:
+- yarn cdn-bundle
+- cd build
+- browser-sync start --cwd -s --http
+
+You should now see output indicating that files are being served from `localhost:3000`
+
+Now switch to the EQ-Questionnaire-Runner Repo
+
+### On EQ-Questionnaire-Runner Repo
+Checkout branch you want to test on, usually `master`
+
+Edit your .development.env with following:
+
+```
+CDN_URL=http://localhost:3000
+CDN_ASSETS_PATH=
+```
+
+Edit the Makefile to remove `build` from the run command. Should now look like this:
```

Comment:
build also does load-schemas and translate, are you just trying to remove load-design-system-templates?

Reply:
I normally just do this but Ill change it to remove `load-design-system-templates` from `build` instead